dask/bytes/local.py

```diff
@@ -87,10 +95,25 @@ def open_files(path):
             for _path in filepaths]
 
 
+def write_files(paths, mode='wb'):
+    """ Open many files for writing.  Return delayed objects.
+
+    See Also
+    --------
+    dask.bytes.core.write_files: User function
+    """
+    myopen = delayed(open)
+    files = [myopen(_path, mode=mode,
+                   dask_key_name='write-%s' % tokenize(_path, time.time()))
+             for _path in paths]
+    return files
```

Comment:
`Bag.to_textfiles` needs something like this.  We can't trust `write_bytes` to work because we can't trust that the data for a particular partition will fit into RAM.  Perhaps the name I'm looking for isn't `write_files` but rather `write_iterator`.